master/internal/api_checkpoint.go

```diff
@@ -93,8 +96,8 @@ func (m *Master) canDoActionOnCheckpointThroughModel(
 			return authz.SubIfUnauthorized(err, nil)
 		}
 	}
-	return status.Error(codes.PermissionDenied,
-		fmt.Sprintf("cannot access checkpoint: %s", id.String()))
+	return apiutils.AddRBACSuffix(status.Error(codes.PermissionDenied,
```

Comment:
IMO a permission-denied coded error is already tagged appropriately and shouldn't need the suffix. probably doesn't hurt though.

Reply:
Resolved -- definitely makes sense to limit `AddRBACSuffix` to only `codes.NotFound`, and I think this was just a lapse from me.